app/services/catalog/add_to_order.rb

```diff
@@ -9,7 +9,11 @@ def initialize(params)
     def process
       order = Order.find_by!(:id => @params[:order_id])
       @order_item = order.order_items.create!(order_item_params)
+
       self
+    rescue ActiveRecord::RecordInvalid => e
+      Rails.logger.error("Error creating order item for order_id #{order.id}: #{e.message}")
+      raise
```

Comment:
@eclarizio @mkanoor do you see any issue with also duplicating these logger error messages in `ProgressMessage`, I realize we'll be able to see them in our logs, but could potentially speed up the troubleshooting process. @gmcculloug suggested this I believe, so didn't want to forget to discuss it.